firstvoices/backend/serializers/import_job_serializers.py

```diff
@@ -0,0 +1,144 @@
+import logging
+
+import tablib
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+from tablib import InvalidDimensions
+
+from backend.models.import_jobs import ImportJob, ImportJobReport, ImportJobReportRow
+from backend.models.media import File
+from backend.permissions.predicates.base import is_superadmin
+from backend.serializers.base_serializers import (
+    CreateSiteContentSerializerMixin,
+    SiteContentLinkedTitleSerializer,
+)
+from backend.serializers.media_serializers import FileUploadSerializer
+from backend.serializers.utils import (
+    get_site_from_context,
+    validate_all_headers,
+    validate_required_headers,
+)
+from backend.serializers.validators import SupportedFileType
+
+CSV_MIME_TYPE = "text/csv"
+
+
+class ImportReportRowSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ImportJobReportRow
+        fields = ["row_number", "status", "errors"]
+
+    row_number = serializers.IntegerField(read_only=True)
```

Comment:
ImportReportRow and ImportReport serializers will be adjusted during development of the next step of the pipeline.